src/commonMain/kotlin/com/charleskorn/kaml/YamlConfiguration.kt

```diff
@@ -52,6 +54,8 @@ public data class YamlConfiguration(
     internal val allowAnchorsAndAliases: Boolean = false,
     internal val yamlNamingStrategy: YamlNamingStrategy? = null,
     internal val codePointLimit: Int? = null,
+    @ExperimentalSerializationApi
+    internal val decodeEnumCaseInsensitive: Boolean = false,
```

Comment:
I don't love this name... how about something like `enableCaseInsensitiveEnumDeserialization`?

Reply:
Hi, @charleskorn. The same naming is used in kotlinx-serialization-json

https://github.com/Kotlin/kotlinx.serialization/blob/master/formats/json/commonMain/src/kotlinx/serialization/json/JsonConfiguration.kt#L36

Also, the current name is short and in the same time - self-descriptive. From my point of view, it would be better to have a name like this rather than something longer (and if we keep the experimental annotation the parameter can be renamed at some point)

If you still would prefer to change the name of the parameter maybe use "Decoding" instead of "Deserialization": `enableCaseInsensitiveEnumDecoding` (it makes the name a bit shorter🙂)

Reply:
If that's the name that kotlinx.serialization uses, then staying consistent with that makes sense to me.